js/banner/banner.encryption.js

```diff
@@ -7,11 +7,21 @@
 ( function ( Banner ) {
 	'use strict';
 
-	var EP;
+	var EP,
+		browserSupportsCryptoAPI = function () {
```

Comment:
As long as I like this check in separate function, and I am not really sure what best Javascript conventions are, but why not declare this function as a "method" (ie. by declaring it as `EP.browserSupportsCryptoAPI = function()`? This seems to make sense to me as it'd be consistent with other parts of "Banner" library but if there are reasons not to make it then great